website/translated_docs/zh-CN/unity-truffle-loom-sample.md

```diff
@@ -1,13 +1,14 @@
 ---
 id: unity-truffle-loom-sample
 title: Unity Truffle Loom 快速上手
-sidebar_label: 快Unity Truffle Loom 快速上手
+sidebar_label: Unity Truffle Loom 快速上手
```

Comment:
you shouldn't be checking in Chinese translations like this, they should go in crowding. also they should start as English

Reply:
It was on hackthon, luke found a medium post and thought it would be helpful for the beginner, so we just translate it to Chinese quickly and publish it. will do it on crowding from now on.